src/System.Console/tests/ConsoleEncoding.cs

```diff
@@ -11,6 +11,8 @@
 
 public partial class ConsoleEncoding : RemoteExecutorTestBase
 {
+    public static bool ConsoleEncodingAvailable => !PlatformDetection.IsUap || PlatformDetection.IsWindows10RS3OrGreater;
```

Comment:
Ah ok. We should verify your conditoin actually triggers on RS3 otherwise it maybe will never fire and we won't know.

I have a fake test named DumpRuntimeInformationToConsole. We could make it write also whether it thinks it's RS3. Then we can see in Helix and CI output. No need to test locally.